KenticoOnboardingApplication/test/KenticoOnboardingApplication.Tests.Base/Comparers/ItemComparer.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using KenticoOnboardingApplication.Contracts.Models;
+
+namespace KenticoOnboardingApplication.Tests.Base.Comparers
+{
+    public sealed class ItemComparer : IEqualityComparer<Item>
+    {
+        private static readonly Lazy<ItemComparer> s_lazyComparer = new Lazy<ItemComparer>(() => new ItemComparer());
+        public static ItemComparer LazyComparer => s_lazyComparer.Value;
```

Comment:
It's usually called `Instance` but that's just a great joke of your fitting the assignment 🤣 😍